app/assets/javascripts/short-form/templates/b2-contact.html.slim

```diff
@@ -4,16 +4,16 @@
       | {{'B2_CONTACT.TITLE' | translate:applicantFirstName()}}
 
   .app-inner.inset.border-bottom
-    .form-group.has-option ng-class="{ error: $scope.requiredContactInformationMissing() }"
+    .form-group.has-option ng-class="{ error: inputInvalid('phone') }"
```

Comment:
right on! I see the ng-required function. Cool! I like the idea of just running that function and marking the input invalid so we only need to run a one function for all the error classes.